release-tools/build.make

```diff
@@ -101,9 +106,42 @@ push-%: container-%
 		fi; \
 	done
 
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
```

Comment:
Introducing another name for `PULL_BASE_REF` IMHO obscures what's going on. Better just use `PULL_BASE_REF` directly below.

Reply:
Removed `RELEASE_ALIAS_TAG`. Using `PULL_BASE_REF` directly in push-multiarch target